ssl/create_certificates.sh

```diff
@@ -0,0 +1 @@
+{\rtf1}
```

Comment:
This looks weird?

Reply:
you are right. looks like the changes did not save correctly. I fixed it now.